Assets/_BForBoss/_VFX/Scripts/PeriodicallyRestartEffect.cs

```diff
@@ -0,0 +1,30 @@
+//-----------------------------------------------------------------------
+// <copyright file="PeriodicallyRestartEffect.cs" company="PERIGON GAMES">
+//     Copyright (c) PERIGON GAMES. 2020 All Rights Reserved
+// </copyright>
+//-----------------------------------------------------------------------
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.VFX;
+
+namespace BForBoss
+{
+    [ExecuteInEditMode]     // Remove this if you don't want the script to update in editor.
+    public class PeriodicallyRestartEffect : MonoBehaviour
+    {
+
+        [SerializeField] private float _wait_time = 1.0f;     // Time between shots.
```

Comment:
We normally go for camelCase with `_` in the beginning. 

https://perigongames.atlassian.net/wiki/spaces/PG/pages/10584065/C+Code+Format
This is the coding format we follow as much as we can

Reply:
Oh yeah that is my bad! I missed those variables names, I'll fix them up.